nRFBlinky/ViewControllers/ScannerView/ScannerTableViewController.swift

```diff
@@ -69,15 +69,19 @@ class ScannerTableViewController: UITableViewController, CBCentralManagerDelegat
     // MARK: - UITableViewDelegate
     
     override func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
-    }
-
-    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         if discoveredPeripherals.count > 0 {
             hideEmptyPeripheralsView()
         } else {
             showEmptyPeripheralsView()
         }
+        return discoveredPeripherals.count > 0 ? 1 : 0
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return "Nearby devices"
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return discoveredPeripherals.count
```

Comment:
collect share (save state_discovered : savePeripheral(s).count)()= survive always supervisor; root check and fix "index" as indexJ;*}  save state save_all ON